src/Components/Card.js

```diff
@@ -20,8 +21,7 @@ class Card extends React.Component {
          <React.Fragment>
             <br></br>
             <h3>{menuItem.item_name}</h3>
-            <h4>{menuItem.price}</h4>
-            <h5>{menuItem.description}</h5>
+            <h5>{menuItem.description}.......{menuItem.price}</h5> 
```

Comment:
This is cool, seems like a more realistic way to depict the menu descriptions and price. Nice!